.gitignore

```diff
@@ -0,0 +1 @@
+output
```

Comment:
`output/**` ?
Guessing you meant the output folder that gets generated from bundle.

Reply:
updated, I think it should have the same effect.